google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/SubscriberSnippets.java

```diff
@@ -66,4 +69,37 @@ public void failed(Subscriber.State from, Throwable failure) {
     subscriber.stopAsync().awaitTerminated();
     // [END startAsync]
   }
+
+  private void createSubscriber() throws Exception {
+    // [START async_pull_subscription]
```

Comment:
Where is this used? I grepped through the google-cloud-java repo and don't see it referenced.